src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/AvoidUninstantiatedInternalClasses.cs

```diff
@@ -158,10 +166,7 @@ public sealed override void Initialize(AnalysisContext context)
 
                     foreach (var type in uninstantiatedInternalTypes)
                     {
-                        if (!hasInternalsVisibleTo || context.Options.GetBoolOptionValue(EditorConfigOptionNames.IgnoreInternalsVisibleTo, Rule, type, context.Compilation, defaultValue: false))
-                        {
-                            context.ReportDiagnostic(type.CreateDiagnostic(Rule, type.FormatMemberName()));
-                        }
```

Comment:
Unintentional change. Will revert shortly (in case someone reviewed this before I revert)